### DISCUSSION TITLE
MacBook Pro (15-inch, 2019) didn't work on big sur 11.4 -- helper v12 not found

### DISCUSSION THREAD

**Discussion by e19he** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#318] [Q&A]
I've tried resetting the SMC, reinstalling the software and enabling root accounts, all to no avail.please help me thanks!

**Comment by MatthiasKerbl** [comment]
Hi @e19he,

an update will come soon which will fix this issue.

**Reply by e19he** [reply]
Thank you
